libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/ClientManager.java

```diff
@@ -389,6 +392,12 @@ public Bundle createNewAccount(String accountName, String username, String refre
             }
         }
         Account acc = new Account(accountName, getAccountType());
+
+        String encodedAdlParams = OAuth2.parameterMapToString(loginOptions.additionalParameters);
```

Comment:
What we store in the account is stuff we got from the server, why would we store the additional parameters if they are only used when calling getAuthorizationUrl?